File: backends/spring-java/src/main/java/dev/stackverse/backend/StackverseBackendApplication.java
package dev.stackverse.backend;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;

@SpringBootApplication
@ConfigurationPropertiesScan
public class StackverseBackendApplication {
    public static void main(String[] args) {
        SpringApplication.run(StackverseBackendApplication.class, args);
    }
}

File: backends/spring-kotlin/src/main/kotlin/dev/stackverse/backend/StackverseBackendApplication.kt
package dev.stackverse.backend

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication
@ConfigurationPropertiesScan
class StackverseBackendApplication

fun main(args: Array<String>) {
    runApplication<StackverseBackendApplication>(*args)
}
